### ISSUE TITLE
Customisation of the type matcher doesn't work

### ISSUE THREAD

**Issue by almogriskified** [bigcommerce/gruf-rspec#7]
The current customisation of the type matcher for Gruf controllers, doesn't seem to work.
There is a value in ENV that looks like it's supposed to affect it, but the value is being reset every time the module is being loaded.

This seems to fix the issue -
change this:
```ruby
def reset
   VALID_CONFIG_KEYS.each do |k, v|
     send((k.to_s + '='), v)
   end
   self.rpc_spec_path = '/spec/rpc/'
   options
end
```

to this:
```ruby
def reset
   VALID_CONFIG_KEYS.each do |k, v|
     send((k.to_s + '='), v)
   end
   self.rpc_spec_path = VALID_CONFIG_KEYS[:rpc_spec_path]
   options
end
```
And then when we want a custom path all we have to do is set ENV['RPC_SPEC_PATH'] before we require 'gruf/rspec'.

For your README:

> ### RSpec Controller Matcher Configuration
> 
> By default, the type matcher for Gruf controllers matches in `/spec/rpc`. You can customize this by overriding the value
> of ENV['RPC_SPEC_PATH'] before the require of 'gruf/rspec' in `spec_helper.rb` like so:
> 
> ```ruby
> ENV['RPC_SPEC_PATH'] = '/spec/rpc_controllers'
> require 'gruf/rspec'
> ```

**Comment by eiei0** [comment]
Is there any update on this? We are experiencing the same thing.

**Comment by splittingred** [comment]
This is now fixed in 0.3.0. Thanks!
